docs/source/index.rst

```diff
@@ -0,0 +1,20 @@
+`idelib`: a Python Library for accessing enDAQ sensor recordings
+==============================================================
```

Comment:
Sphinx generates a warning that the underscore line is too short, which the tests are counting as an error. @CFlaniganMide Can the tests be configured to let warnings be warnings?

Reply:
tbh it takes a couple minutes to write, commit and upload fixes for formatting issues like this; I prefer this behavior and would like this left as-is

Reply:
but to answer your question: yes, warnings can be left as warnings by removing the `-W` flag from the build command.

Reply:
formatting fixed in 65a79aa993e7c14739a4064c41538991e9bdd2ed